foundry.toml

```
@@ -26,3 +27,6 @@ sepolia = { key = "${ETHERSCAN_API_KEY}" }
 line_length=100
 bracket_spacing=true
 wrap_comments=true
+
+[profile.ci]
+fuzz = { run = 50 }
```

```suggestion
fuzz = { runs = 50 }
```

Should we run a bit more?

I also noticed that the inline comments seems to take precedence over setting it via env var or in foundry.toml which seems weird. In that case should we not use the inline comments?

---

I use inline ones previously because some tests run slower than others, but now the released built `diff-test` binary, all of these tests are fast, so I will remove any inline configs for fuzzer (just use 256 runs by default) and even remove this as well (256 for ci seems good enough)?

---

Or we could add something like the following and run it once a day on the CI.

```
[profile.fuzz]
fuzz = { runs = 10000 }
```

---

> I use inline ones previously because some tests run slower than others, but now the released built `diff-test` binary, all of these tests are fast, so I will remove any inline configs for fuzzer (just use 256 runs by default) and even remove this as well (256 for ci seems good enough)?

Yeah if that's not too slow that is fine I think.

Since it's almost free and only costs CPU time I think it's not a bad idea to run more iterations on the CI occasionally, but we don't have to do it as part of this PR.

---

done in [90bf8d1](https://github.com/EspressoSystems/espresso-sequencer/pull/722/commits/90bf8d1c42a6613213ef9efe4711a57e3c803a89)